.happy/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "config_version": "v1",
+    "terraform_version": "0.13.5",
+    "default_env": "rdev",
+    "app": "hosted-cellxgene",
+    "default_compose_env": ".env.ecr",
+    "environments": {
+        "rdev": {
```

Comment:
Do we not need `terraform_directory` here? (just comparing to `config.json` for portal)

Reply:
Ah, nm, that's in the infra repo. 🤦

Reply:
Right, `happy` script doesn't need terraform stuff here since we're just calling `push` subcmd to build the docker image.  I really don't like having any `happy` stuff in this repo at all, and we can eliminate all this with an alternate rdev deployment design: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/253#issuecomment-967169708